home-open-api/src/main/resources/yaml/openapi.yaml

```diff
@@ -498,7 +498,132 @@ paths:
           $ref: '#/components/responses/NotFound'
         default:
           $ref: '#/components/responses/InternalServerError'
-
+  /cooperation/{id}/tasks:
+    post:
+      tags:
+        - cooperation
+      summary: TODO
+      description: TODO
+      operationId: createTask
+      parameters:
+        - $ref: '#/components/parameters/p_id'
+      requestBody:
+         $ref: '#/components/requestBodies/TaskTypeBody'
+      responses:
+        201:
+          $ref: '#/components/responses/TaskResponse'
+        400:
+          $ref: '#/components/responses/BadRequest'
+        401:
+          $ref: '#/components/responses/Unauthorized'
+        403:
+          $ref: '#/components/responses/Forbidden'
+        default:
+          $ref: '#/components/responses/InternalServerError'
+    get:
+      tags:
+        - cooperation
+      summary: TODO
```

Comment:
Please do not leave TODOs in design. Can you imagine a plan for a building with TODOs for the height and width?